README.md

```diff
@@ -126,6 +126,11 @@ In that case you should still pass network name with `-n` flag, especially when
 If `SETH_KEYFILE_PATH` is not set then client will create X ephemeral keys (60 by default, configurable) and won't return any funds.
 Use `SETH_KEYFILE_PATH` for testnets/mainnets and `ephemeral` mode only when testing against simulated network.
 
+In case your RPC node requires custom headers you can set them using `SETH_RPC_HEADERS` in `k=v,k=v` format
+```
+export SETH_RPC_HEADERS=Host=http.com,Host=http.com
```

Comment:
@skudasov I wonder if you want people to use ~/.testsecrets to load it this? If yes, you need to update `config.ReadFromEnvVar()` in CTF. Another question - do we want custom headers per network?

Reply:
@skudasov btw, `Host=http.com,Host=http.com` this format does not allow for proper masking on Github CI. We use standard key=value format and mask each value in a `.env` file. Here is the [github step that does it](https://github.com/smartcontractkit/chainlink-github-actions/blob/main/chainlink-testing-framework/run-tests/action.yml#L302)

Reply:
This is infra stuff, no, I want to use our vault or GitHub Secrets directly. People won't know these headers so there is nothing to override.

Reply:
ok, so if this is not part of Test Secrets then in this case that's fine